CONTRIBUTING.md

```diff
@@ -9,3 +13,65 @@
 - [Visual Studio Code](https://code.visualstudio.com/)
 - [Microsoft Graph PowerShell SDK](https://github.com/microsoftgraph/msgraph-sdk-powershell), cloned into the same parent folder of this repository. This dependency is only required if you want to generate SDKs for Microsoft Graph.
 - [reportgenerator](https://www.nuget.org/packages/dotnet-reportgenerator-globaltool), if you want to be able to generate coverage reports from the pre-configured visual studio code test tasks.
+
+## Building the project
+
+```sh
+dotnet restore ./src/kiota
+dotnet build ./src/kiota
```

Comment:
```suggestion
dotnet workload restore
dotnet restore
dotnet build
```

Should get everything in order

Reply:
I get an error attempting to run `dotnet workload restore`:
```
>dotnet workload restore
Installing workloads: wasm-tools
Inadequate permissions. Run the command with elevated privileges.
```

Reply:
```suggestion
# when on linux
sudo dotnet workload restore
# when on window, run the following command in an elevated prompt
dotnet workload restore
# for everyone
dotnet restore
dotnet build
```